github-checks-app/github-checks-app-on-enterprise-server.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import sys
+from secrets import token_urlsafe
+from urllib.parse import quote_plus
+
+
+def init_logger():
+    logger = logging.getLogger('Creating GitHub Enterprise Server App')
+    logger.setLevel(logging.INFO)
+    h = logging.StreamHandler(sys.stdout)
+    logger.addHandler(h)
+
+    return logger
+
+
+logger = init_logger()
+
+
+if __name__ == '__main__':
+    domain = input('Enter your GitHub Enterprise Server domain name (i.e. github.companyname.com): ')
+
+    domain = domain.split('://')[-1]
+    domain = domain.split('/')[0]
+
+    callback_url = f'https://blubracket.blubracket.com/api/github_app/auth?domain={domain}'
+    webhook_url = f'https://blubracket.blubracket.com/api/github_app/events?domain={domain}'
```

Comment:
now that these paths are visible we should ensure that they are rate-limited? @malikoth

Reply:
They are.  :)